### DISCUSSION TITLE
Feature Request: Install Extensions via PIP

### DISCUSSION THREAD

**Discussion by CzechJiri** [duckdb/duckdb#11893] [Ideas]
Extensions currently follow custom installation instead of traditional PIP route

- many firms use proxy to download any kinds of software, pip packages are usually consistently inspected and registered
- JIT use (autoload on first use) or custom pre-download is inconsistent with PIP driven dependency management including version control

would it be possible to provide pip driven installation for extensions?

**Comment by Tishj** [comment]
Thanks, this has come up before, we have looked into supporting this but it's not trivial
Please look through the existing discussions as I can recall at least two other discussions that are opened about this 👍

**Comment by wernerdaehn** [comment]
Isn't this exactly what is needed?
https://github.com/santosh-d3vpl3x/duckdb_extensions
